File: tests/fixtures/accuracy.java
package com.example;

import java.util.List;

interface Drawable {
    void draw();
}

class Calculator {
    public int add(int a, int b) {
        return a + b;
    }

    public int compute() {
        return this.add(3, 4);
    }
}

class Circle implements Drawable {
    private int radius;

    public Circle(int radius) {
        this.radius = radius;
    }

    @Override
    public void draw() {
        System.out.println("Drawing");
    }

    public static Circle unit() {
        return new Circle(1);
    }
}

File: tests/fixtures/accuracy.kt
package com.example

import kotlin.math.sqrt

interface Drawable {
    fun draw()
}

abstract class Shape : Drawable {
    abstract fun area(): Double
}

class Calculator {
    fun add(a: Int, b: Int): Int {
        return a + b
    }
}

class Circle(val radius: Double) : Shape() {
    override fun draw() {
        println("Circle")
    }

    override fun area(): Double {
        return Math.PI * radius * radius
    }

    companion object {
        fun unit() = Circle(1.0)
    }
}

fun main() {
    val calc = Calculator()
    println(calc.add(3, 4))
}
